pyuvdata/utils.py

```diff
@@ -1978,7 +1978,7 @@ def calc_uvw(
     from_enu : boolean
         Set to True if uvw_array is expressed in ENU coordinates. Default is False.
     to_enu : boolean
-        Set to True if you would like the output expressed in EN coordinates. Default
+        Set to True if you would like the output expressed in ENU coordinates. Default
```

Comment:
Heh, spelling errors seem to be a regular part of my code -- thanks for catching that!